docs/source/format/Columnar.rst

```diff
@@ -350,6 +352,53 @@ will be represented as follows: ::
     |----------------|-----------------------|
     | joemark        | unspecified (padding) |
 
+.. versionadded:: Arrow Columnar Format 1.4
+
+Variable-size Binary View Layout
```

Comment:
... and indeed here this is described as a different layout.

Reply:
I would agree that this should at least be made consistent with the decision made above (perhaps just added as a subsection within Variable-size Binary).